docker/netbird/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  netbird-client:
+    image: netbirdio/netbird:latest
+    container_name: netbird-client
+    cap_add:
+      - NET_ADMIN
+    environment:
+      - NB_SETUP_KEY=${NB_SETUP_KEY}
```

Comment:
Should this directory also have a .env.example file that includes `NB_SETUP_KEY`?